google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubTest.java

```diff
@@ -592,6 +594,56 @@ public void testReadChangeStreamWaitTimeoutIsSet() throws Exception {
     }
   }
 
+  @Test
+  public void testBatchMutationsPartialFailure() throws InterruptedException {
```

Comment:
Its really hard to see what is being tested.  I see the setup for the test and I see code that triggers the mocked behavior  (which oddly comes beforecausing an unnecessary race). But I dont see what anything being inspected nor do I see anything being verified.
In fact this seems to show a bug, as I would expect batcher.close() to throw an exception

Reply:
Fixed